runner/aws_lambda_scripts/build_push_ecr.sh

```diff
@@ -0,0 +1,121 @@
+# MIT License
```

Comment:
Long bash files are very difficult to maintain in the long run. Can we please have this in python or golang if possible?

Reply:
Python would have been my choice for such scripts. However, since I use the _aws_ command line tool (_awscli_) in these scripts, I went with bash scripting. I am not aware of any python alternatives for the functionalities provided by _awscli_ (even if there are, I don't think it would be as [well-documented](https://docs.aws.amazon.com/cli/latest/reference/index.html#cli-aws) as _awscli_) and running bash commands via repeated `system(...)` calls in python is hardly a clean way to do this. 
Therefore, while I agree with the contention about bash files being hard to maintain, I think bash would be the better choice in this particular case.

Reply:
system() calls would be a bad idea. However, you don't need awscli wrappers. Boto3 is the python library maintained by AWS has complete support.